www/source/blog/2018-03-21-0550-Release.html.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Habitat 0.55.0 Released
+date: 2018-03-21
+author: Elliott Davis
+tags: release notes
+category: product
+classes: body-article
+---
+
+# Habitat 0.55.0 Release Notes
+
+We are happy to announce the release of Habitat 0.55.0. We have a number of new features as well as bug fixes, so please read on for all the details. If you just want the binaries, head on over to [Install Habitat](https://www.habitat.sh/docs/using-habitat/#install-habitat). Thanks again for using Habitat!
+
+## Highlights:
+
+* [Feature] Ability to create and upload secrets via the hab cli
+* [Feature] Improve startup time of Windows container studios
+* [Feature] Windows container Studio runs its background supervisor as a service
+* [Bugfix] Windows Supervisor honors a Plan's pkg_svc_user
+* [Bugfix] Windows container Studio correctly streams Supervisor output from Get-SupervisorLog
+* [Bugfix] Updated default.toml files are seen on package update (fixes https://github.com/habitat-sh/habitat/issues/4576)
```

Comment:
It might be nicer just to link these two fixes (like "fixes [#4576](https://github.com/habitat-sh/habitat/issues/4576)" for example).